pkg/corecli/extract.go

```diff
@@ -0,0 +1,62 @@
+package corecli
+
+import (
+	"github.com/dcos/dcos-cli/pkg/fsutil"
+	"github.com/dcos/dcos-cli/pkg/plugin"
+	"github.com/pelletier/go-toml"
+	"github.com/spf13/afero"
+	"path/filepath"
+)
+
+// TempPlugin will read the bundled corecli into a memmapfs and return a fake plugin from that data.
+func TempPlugin() (*plugin.Plugin, error) {
```

Comment:
Can we leave this hidden as an implementation detail instead of exposing a new package? See https://golang.org/doc/go1.4#internalpackages